app/src/main/java/org/wikiedufoundation/wikiedudashboard/ui/welcome/OutreachProgramsDashboardFragment.kt

```diff
@@ -60,9 +60,9 @@ class OutreachProgramsDashboardFragment : Fragment() {
         savedInstanceState: Bundle?
     ): View? {
         val view = inflater.inflate(R.layout.fragment_outreach_programs_dashboard, container, false)
-        cvSignUpWikipedia = view.findViewById(R.id.cv_signup_wikipedia)
-        cvSignUpWikipedia = view.findViewById(R.id.cv_login_wikipedia)
-        cvLoginWikipedia = view.findViewById(R.id.cv_login_wikipedia)
+        cvSignUpWikipedia = view.findViewById(R.id.cvSignUpWikipedia)
+        cvSignUpWikipedia = view.findViewById(R.id.cvLoginWikipedia)
+        cvLoginWikipedia = view.findViewById(R.id.cvLoginWikipedia)
```

Comment:
I guess we don't need to use findViewById . It will directly take the components from XML.

Reply:
This PR is for wikiEducationDashboardFragment.kt